content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -178,8 +166,8 @@ after v1.7. Please update existing applications to use the fields directly.
 `service.beta.kubernetes.io/external-traffic` annotation has a different set of values
 compared to the `service.spec.externalTrafficPolicy` field. The values match as follows:
 
-* "OnlyLocal" for annotation <-> "Local" for field
-* "Global" for annotation <-> "Cluster" for field
+* OnlyLocal for annotation <-> Local for field
```

Comment:
```suggestion
* OnlyLocal for annotation ⭤ Local for field
```

Reply:
However: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#feature-availability is outdated. It'd be fine to drop this entire fragment.